src/pages/resultsView/querySummary/QuerySummaryUtils.tsx

```diff
@@ -64,18 +64,18 @@ export function getAlterationSummary(
     }
 }
 
-export function getStudyViewFilterHash(
+export function submitToStudyViewPage(
+    queriedStudies: Pick<CancerStudy, "studyId">[],
     samples: Pick<Sample, "sampleId" | "studyId">[],
     hasVirtualStudies: boolean,
     sampleLists?: Pick<SampleList, "category">[]) {
     const hasAllCaseLists = _.some(sampleLists, sampleList => sampleList.category === 'all_cases_in_study');
-    let studyViewFilterHash: string | undefined;
     if (samples.length > 0 && (hasVirtualStudies || !hasAllCaseLists)) {
         const sampleIdentifiers = samples.map(sample => ({
             sampleId: sample.sampleId,
```

Comment:
If we are going to remove studyIds later on in study view, we better use uniqueSampleId here

Reply:
This is equivalent of `sampleIdentifiers` in `StudyViewFilter`, which doesn't support `uniqueSampleIds`

Reply:
Oh right, I misunderstood the structure. Sorry about that.